tweets/2022-03/meetup8.tweet

```diff
@@ -0,0 +1 @@
+Want to drive your community and make business-oriented decisions using those open source-ecosystem data sets? Join us on March 22nd, 11:00ET for the Operate First Data Science Meetup. Mark your calendars now https://calendar.google.com/calendar/u/2?cid=N3QyMm1ydm92amNmdTZqZm5ucDRuMmtkZTRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ
```

Comment:
```suggestion
Want to drive your community and make business decisions using open source-ecosystem data sets? Join us on March 22nd, 11:00ET for Operate First Data Science Meetup 8 as @calidolphinn & James Kunstle share discoveries from the SanDiego project. Mark your calendars https://calendar.google.com/calendar/u/2?cid=N3QyMm1ydm92amNmdTZqZm5ucDRuMmtkZTRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ
```

Reply:
@oindrillac - As much as I would love to include their names, it's going beyond the word limit :disappointed:

Reply:
@aakankshaduggal what is the word limit? Or is it a character limit? https://github.com/operate-first/operate-first-twitter/blob/main/tweets/2022-03/meetup7.tweet seemed to accept 50 words

Reply:
It's a character limit. This proposed tweet is 8 characters more than expected.

Reply:
Try above. Tested it using the UI, twitter seems to accept it.

Reply:
yes, works :+1: